quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/utils/DocumentationUtils.java

```diff
@@ -71,19 +71,12 @@ public static MarkupContent getDocumentation(ExtendedConfigDescriptionBuildItem
 		// Config Phase
 		addParameter("Phase", getPhaseLabel(item.getPhase()), documentation, markdown);
 
-		// Location
-		String location = item.getLocation();
-		if (location != null) {
-			int slashIndex = location.lastIndexOf('/');
-			if (slashIndex != -1) {
-				location = location.substring(slashIndex + 1, location.length());
-			}
-			addParameter("Location", location, documentation, markdown);
+		// Extension name
+		String extensionName = item.getExtensionName();
+		if (extensionName != null) {

```

Comment:
Can we also just do `addParameter("Extension", item.getExtensionName(), documentation, markdown);`, since `addParameter()` checks for null value [here](https://github.com/angelozerr/quarkus-ls/blob/5ef83b4f8e338d15200676567d05b58592060e4a/quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/utils/DocumentationUtils.java#L97)?

Reply:
Good catch! It should be fixed now.